ubuntu_20.04-dev.dockerfile

```diff
@@ -9,7 +9,7 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 # install apt dependencies
 RUN apt-get -y  update
 RUN apt-get install -y software-properties-common \
-                       python3-pip \
+                       python-is-python3 \
```

Comment:
We prefer to use `update-alternatives` as a robust way to accomplish this across many variations